airflow/operators/python.py

```diff
@@ -518,7 +518,13 @@ def __init__(
         self.python_version = python_version
         self.system_site_packages = system_site_packages
         self.pip_install_options = pip_install_options
-        self.skip_exit_code = skip_exit_code
+        self.skip_on_exit_code = (
+            skip_on_exit_code
+            if isinstance(skip_on_exit_code, list)
```

Comment:
Would be better to do `isinstance` against either `int` or `collections.abc.Iterable` instead, to allow e.g. `skip_on_exit_code=(1, 2)`. No reason to restrict ourselves to just lists here.

Reply:
I like the idea of the `Container` type, usually I use `Iterable`, but `Container` is more suitable since we are using `__contains__` and not `__iter__`